examples/src/main/java/kafka/examples/KafkaProperties.java

```diff
@@ -17,7 +17,6 @@
 package kafka.examples;
 
 public class KafkaProperties {
-    public static final String TOPIC = "topic1";
```

Comment:
can you please look into when this was added and why it wasn't used? Unused variables can sometimes hint towards bugs where we are not using variables when we should.

Reply:
@divijvaidya  Thank you so much for the review

When this class is created on it's earlier version, it stored all related properties needed to run all examples this is an example of what is was when created 

```
  final static String zkConnect = "127.0.0.1:2181";
  final static  String groupId = "group1";
  final static String topic = "topic1";
  final static String kafkaServerURL = "localhost";
  final static int kafkaServerPort = 9092;
  final static int kafkaProducerBufferSize = 64*1024;
  final static int connectionTimeOut = 100000;
  final static int reconnectInterval = 10000;
  final static String topic2 = "topic2";
  final static String topic3 = "topic3";
  final static String clientId = "SimpleConsumerDemoClient";**
```

Then it was cleaned, and intended to keep configuration needed to run Kafka,

Reply:
@divijvaidya  I think even the two variables needs to be refactored since they are used only once and for the exactly once semantic example and can be replaced by one variable the contains the bootstrapServers instead of concatenating three strings to build a boostrapServers url 


```
    public static final String KAFKA_SERVER_URL = "localhost";
    public static final int KAFKA_SERVER_PORT = 9092;
```


I think I can update this PR cover this point

Reply:
@divijvaidya  I updated my pull request and made some suggestions like reusing `BOOTSTRAP_SERVERS`  property in all examples when it's needed 

Please can you review it again